xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -35,14 +35,12 @@
 
 #include "DRMUtils.h"
 
-static struct drm *m_drm = nullptr;
-
 static drmModeResPtr m_drm_resources = nullptr;
 static drmModeCrtcPtr m_orig_crtc = nullptr;
 
-struct drm *CDRMUtils::GetDrm()
+CDRMUtils *CDRMUtils::GetDrm()
 {
-  return m_drm;
+  return this;
 }
```

Comment:
IMHO this is what you most likely not want to do.
I don't see any use of this getter, so I'd suggest to remove it.

Reply:
leftovers, will remove.